aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -108,13 +106,36 @@ http://localhost/Speaker/Evaluations#SpeakerEvaluations
 ```
 Hash tags are useful when doing client side applications. They can be used for easy marking and searching in JavaScript for example.
 
+- - -
+
+### asp-area
+
+```asp-area``` sets the area name that ASP.NET core will use to set the appropriate route with.  Below are examples of how the area attribute causes a remapping of routes.  That is, by setting ```asp-area``` to Blogs will invoke the Areas functionality such that the directory Areas/Blogs will prefix the associated controllers and views for this anchor tag.
 
-## asp-area
+* Project name
 
-```asp-area``` allows for a razor view area to be associated with the hyperlink URL. That is, if you have a project setup in a similar fashion to the project pictured below, the link generated will include as its first segment the area you mention.
+  * wwwroot
```

Comment:
Make *wwwroot* italics. Same below (I think) for any proper folder or file. Usually, they keep folders and files in italics wherever they appear in docs, including when they appear in lists and tables. @tdykstra will correct me if I'm mistaken.